src/components/Page/components/Header/Header.scss

```diff
@@ -98,4 +107,35 @@ $action-menu-rollup-computed-width: icon-size() + (spacing(tight) * 2);
   }
 
   @include print-hidden;
+
+  &.newDesignLanguage {
+    margin-top: 0;
+  }
+}
+
+.Row {
```

Comment:
`.Row`, `.LeftAlign`, and `.RightAlign` are the extent of the new page layout classes. It consists of two rows, both rows having left and right aligned content. That’s it.